workers/unity/Assets/Playground/Scripts/Player/ConnectButtonClickSystem.cs

```diff
@@ -0,0 +1,99 @@
+using Improbable.Gdk.Core;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.UI;
+#if UNITY_ANDROID
+using Improbable.Gdk.Android;
+
```

Comment:
newline

Reply:
Rider complains if there is no newline. Should I just overrule it?

Reply:
just saw that we did the same thing for other definitions.. slightly weird ,but leave it for now

Reply:
Turn off the warning in rider.
Pu a newline above the #if though.

Reply:
Resolved offline. Leaving it like that to be consistent with other files.